.changeset/fair-apricots-change.md

```diff
@@ -0,0 +1,6 @@
+---
+"@sveltejs/adapter-cloudflare": minor
+"@sveltejs/adapter-cloudflare-workers": minor
+---
+
+feat: atomic builds
```

Comment:
I think this may be a patch fix instead that prevents those transient error messages during rebuilds.